src/components/modal/components/Profile/utils.ts

```diff
@@ -68,17 +74,16 @@ export function buildProfileFromMetadata(
           {
             ...coverPicture,
             src: coverPicture.url,
-            fit: "fit" in coverPicture ? coverPicture.fit : undefined,
+            fit:
+              "fit" in coverPicture ? (coverPicture.fit as string) : undefined,
```

Comment:
what is the type of coverPicture.fit? I dont like forcing casts if we can avoid them

Reply:
coverPicture.fit type is `string`
![image](https://github.com/user-attachments/assets/da4061da-327c-40fc-81f1-21176f8321fb)